pkgs/applications/editors/idea/default.nix

```diff
@@ -340,12 +340,12 @@ in
 
   datagrip = buildDataGrip rec {
     name = "datagrip-${version}";
-    version = "2016.3.2";
+    version = "2016.3";
```

Comment:
no. leave this pull request open. You can added changes to your existing commit using `git commit --amend` and a `git push --force origin update-jetbrains`